browsers/samsunginternet_android.json

```diff
@@ -143,15 +143,21 @@
         },
         "10.0": {
           "release_date": "2019-08-22",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "71"
         },
         "10.2": {
           "release_date": "2019-10-09",
-          "status": "beta",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "71"
+        },
+        "11.0": {
+          "release_date": "2019-10-22",
```

Comment:
Given https://github.com/mdn/browser-compat-data/pull/5318#issuecomment-566157046, would you switch this to a plausible date in December? Thank you!

Reply:
Sure thing, done!  Does the new date seem reasonable?